### DISCUSSION TITLE
v0.15.4-dev

### DISCUSSION THREAD

**Discussion by sriramveeraghanta** [makeplane/plane#3701] [Announcements]
A swift and sleek release where we introduce ARM builds and address bugs in Issue views, delivering enhanced performance following our v0.15-dev release.

### New things

- ARM builds are now accessible on Docker Hub. Check them out [here](https://hub.docker.com/repositories/makeplane). 🎉

### Bug Fixes

- Inbox stability has been improved. We've resolved issues such as auto-saves and loading states when adding issues from the Inbox.
- The pesky `undefined` alert that popped up when using the copy issue link button has been fixed. (#3686)
- Custom theme colour configuration receives a minor makeover for a smoother experience. (#3691)

<hr /><em>This discussion was created from the release <a href='https://github.com/makeplane/plane/releases/tag/v0.15.4-dev'>v0.15.4-dev</a>.</em>

**Comment by olivierlambert** [comment]
Quick report: just after doing the update on this release vs the previous one (self-hosted via the Docker compose script with "upgrade"), I have now:
```
Exception Detected!

We're Sorry! An exception has been detected, and our engineering team has been notified. 
```

Is there anything I would have missed?

edit: this is only happening on the home/dashboard view (the rest is working), so it feels like a bug :)
